.github/workflows/test.yml

```diff
@@ -7,10 +7,13 @@
 
 name: Continuous Integration
 
-# Only trigger the workflow when pushing to master
 on:
```

Comment:
I've changed this from a list to a dict to follow the schema from https://json.schemastore.org/github-workflow (otherwise VSCode complains).